src/swagger/handleJson.js

```diff
@@ -0,0 +1,57 @@
+import { get } from "lodash";
+import Field from "../Field";
+import Resource from "../Resource";
+
+String.prototype.capitalize = function() {
```

Comment:
It's confusing because it doesn't do the same thing than underscore/lodash capitalize.

Can you use [`_.upperFirst()`](https://lodash.com/docs/4.17.10#upperFirst) instead of adding new functions to the global String prototype? We already use lodash in the project: https://github.com/api-platform/api-doc-parser/blob/master/package.json#L39